cute_tiled.h

```diff
@@ -2066,9 +2066,8 @@ cute_tiled_object_t* cute_tiled_read_object(cute_tiled_map_internal_t* m)
 			break;
 
 		case 1485919047363370797U: // class
-			CUTE_TILED_WARNING("Class field of Tiled objects is not yet supported. Ignoring field.");
-			while (cute_tiled_peak(m) != ',' && cute_tiled_peak(m) != '}') cute_tiled_next(m);
-			if (cute_tiled_peak(m) == '}')	continue;
+			// This is technically different than type, but it used the same way in newer versions of Tiled
+			cute_tiled_intern_string(m, &object->type);
```

Comment:
Seems strange to load `class` into `type`. Should we not have a `type` case that would load `type`? While the [JSON Map Format](https://doc.mapeditor.org/en/stable/reference/json-map-format/) documents both types as strings, it seems weird to mix the two.

Reply:
What better way to represent a class, than as a string? I think I recall entire objects can be stored there, but, for now at least parsing the string is better than nothing.

Reply:
As a strong makes sense. I'm more concerned about `class` being loaded into `type`.

Reply:
I added more notes on #362 somehow `class` is sometimes used instead of `type`. it's the same data, but in some situations it will name it one or the other.

Reply:
Ah, that makes sense. Thanks for some of the context :+1:..... Since `class` isn't a property associated with `object`, seems like this is a fine addition to have.